packages/qwik/src/core/qrl/qrl-class.ts

```diff
@@ -159,7 +159,9 @@ export const createQRL = <TYPE>(
     $captureRef$: captureRef,
     dev: null,
   };
-  const qrl = Object.assign(invokeQRL, methods);
+  Object.setPrototypeOf(methods, Function.prototype)
+  seal(methods)
+  const qrl = Object.setPrototypeOf(invokeQRL, methods);
```

Comment:
Why do we have to do this? This has a big performance penalty.: https://stackoverflow.com/questions/52654781/why-does-using-object-setprototypeof-impact-performance-heavily